airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java

```diff
@@ -95,9 +106,8 @@ public JsonNode toJdbcConfig(final JsonNode config) {
     // and can't
     // remove zero date values.
     // since zero dates are placeholders, we convert them to null by default
-    jdbcUrl.append("?zeroDateTimeBehavior=convertToNull");
     if (!additionalParameters.isEmpty()) {
-      jdbcUrl.append("&");
+      jdbcUrl.append("?");
```

Comment:
append "?" instead of "&" since this `zeroDateTimeBehavior=convertToNull` is already included in `additionalParameters`